### DISCUSSION TITLE
Pakistan is not in the list of 2FA SMS countries.

### DISCUSSION THREAD

**Discussion by abubakarfarooq13** [community/community#137609] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Are you mad Github

**Comment by ebndev** [comment, marked as answer]
Hey there @abubakarfarooq13,

Thanks so much for reaching out in GitHub's Community! If your country is not on [the list](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/countries-where-sms-authentication-is-supported#supported-countries-for-sms-authentication), then we aren't currently able to reliably deliver text messages to your country. We track delivery rates across all countries, including ones not in the list, and review it to understand when we have to remove countries or can add them back. We do not add countries with low deliverability rates, because it leads to users being locked out of their accounts when they can’t receive the SMS for 2FA.

If GitHub doesn’t support two-factor authentication via text message for your country of residence, you must set up authentication via a TOTP application. For more information on how to configure 2FA, please review our [documentation](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/configuring-two-factor-authentication#configuring-two-factor-authentication-using-a-totp-mobile-app).

We put together a [FAQ](https://github.com/orgs/community/discussions/68240) in case you have any additional questions that may be answered there!
